deploying dictionary guide (#24)
subquery/subql-dictionary: README.md

# Understanding the dictionary

Visit [How the dictionary works](https://doc.subquery.network/tutorials_examples/howto.html#how-does-a-subquery-dictionary-work) for more information.

# Adding your own dictionary

> 1. You can fork the repository

Reviewer: ```suggestion
1. Fork the repository.
```

> 2. Add your own dictionary project to the root directory

Reviewer: ```suggestion
2. Add your own dictionary project to the root directory.
```

> 3. To publish onto our hosting services, you can use the Github actions

Reviewer: ```suggestion
3. To publish on our Managed Services, you can use the GitHub actions. 
```

>    1. Set a Github environment `DEPLOYMENT`

Reviewer: ```suggestion
   1. Set a GitHub environment to `DEPLOYMENT`.
```

>    2. Set a Github secret named `SUBQL_ACCES_TOKEN` (Your token generated on https://project.subquery.network/ ) under the previously set environment 

Reviewer: ```suggestion
   2. Set a GitHub secret name to `SUBQL_ACCES_TOKEN` (Your token generated on https://project.subquery.network/ ) under the previously set environment.
```

>    3. Create a project using the UI or the CLI

Reviewer: ```suggestion
   3. Create a project using the UI or the CLI.
```

>    4. Navigate to Github actions in your forked repository, select the workflow `Deploy to Hosted Service`

Reviewer: ```suggestion
   4. Navigate to GitHub actions in your forked repository and select the workflow `Deploy to Hosted Service`.
```

>       1. Enter your dictionary directory name in the input field `Dictionary directory name` e.g.`acala`

Reviewer: ```suggestion
      1. Enter your dictionary directory name in the input field `Dictionary directory name` e.g.`acala`.
```

>       2. For the input field `Project Name` enter the name of the project created earlier (If your project name uses spaces please replace them with `-` e.g. `perfect - dictionary` -> `perfect---dictionary`)

Reviewer: ```suggestion
      2. For the input field `Project Name` enter the name of the project created earlier (If your project name uses spaces please replace them with `-` e.g. `perfect - dictionary` -> `perfect---dictionary`).
```
